static/scripts/rewards/web3/use-rpc-handler.ts

```diff
@@ -0,0 +1,17 @@
+import { RPCHandler } from "@keyrxng/rpc-handler/dist/esm/src/rpc-handler";
+import { HandlerConstructorConfig } from "@keyrxng/rpc-handler/dist/esm/src";
+import { AppState } from "../app-state";
+
+export async function useRpcHandler(app: AppState) {
+  const config: HandlerConstructorConfig = {
+    networkId: app.networkId ?? app.reward.networkId,
```

Comment:
Seems like a bad approach. We should abstract the logic inside of the app.networkId getter to do this instead of redundantly writing `app.networkId ?? app.reward.networkId`